components/style/sharing/mod.rs

```diff
@@ -146,16 +167,18 @@ impl<E: TElement> StyleSharingCandidate<E> {
         self.validation_data.pres_hints(*self.element)
     }
 
-    /// Get the classlist of this candidate.
+    /// Compute the bit vector of revalidation selector match results
+    /// for this candidate.
     fn revalidation_match_results(
         &mut self,
         stylist: &Stylist,
-        bloom: &BloomFilter,
+        bloom: &StyleBloom<E>,
     ) -> &BitVec {
         self.validation_data.revalidation_match_results(
             *self.element,
             stylist,
             bloom,
+            false,
```

Comment:
ditto re. the comment on the bool arg. Alternatively, you can use a binary enum like:

```rust
enum BloomFilterValidity {
    KnownValid,
    MaybeInvalid,
}
```

Or something like that.